### ISSUE TITLE
porting to mxnet problem

### ISSUE THREAD

**Issue by pribadihcr** [aiff22/DPED#9]
Hi @aiff22,

I am trying to port your code to MxNet. But there is an issue about loss function definition. 
For example I can not find the corresponding  ops like tf.image.rgb_to_grayscale in MxNet.

Maybe you can guide me. Thanks

**Comment by aiff22** [comment]
Hi @pribadihcr,

While we don't have experience of using MxNet, it looks like some Tensorflow functions are not implemented there. You can solve this problem by implementing them yourself, which in case of ```rgb_to_grayscale``` is as follows (python code):

```def rgb_to_grayscale(rgb):```
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;```return np.dot(rgb[...,:3], [0.299, 0.587, 0.114])```
